static/app/views/settings/projectGeneralSettings.spec.jsx

```diff
@@ -70,74 +78,57 @@ describe('projectGeneralSettings', function () {
+    expect(getField('textbox', 'Name')).toHaveValue('Project Name');
+    expect(getField('textbox', 'Subject Prefix')).toHaveValue('[my-org]');
+
+    // Step 19 of the auto resolve slider equates to 48 hours. This is
+    // different from thee actual field value (which will be 48)
+    expect(getField('slider', 'Auto Resolve')).toHaveValue('19');
+
+    expect(getField('textbox', 'Allowed Domains')).toHaveValue(
       'example.com\nhttps://example.com'
     );
-    expect(wrapper.find('Switch[name="scrapeJavaScript"]').prop('isDisabled')).toBe(
-      false
-    );
-    expect(wrapper.find('Switch[name="scrapeJavaScript"]').prop('isActive')).toBeTruthy();
-    expect(wrapper.find('Input[name="securityToken"]').prop('value')).toBe(
-      'security-token'
-    );
-    expect(wrapper.find('Input[name="securityTokenHeader"]').prop('value')).toBe(
-      'x-security-header'
-    );
-    expect(wrapper.find('Switch[name="verifySSL"]').prop('isActive')).toBeTruthy();
+    expect(getField('checkbox', 'Enable JavaScript source fetching')).toBeChecked();
+    expect(getField('textbox', 'Security Token')).toHaveValue('security-token');
+    expect(getField('textbox', 'Security Token Header')).toHaveValue('x-security-header');
+    expect(getField('checkbox', 'Verify TLS/SSL')).toBeChecked();
   });
 
   it('disables scrapeJavaScript when equivalent org setting is false', function () {
     routerContext.context.organization.scrapeJavaScript = false;
-    wrapper = mountWithTheme(
+    render(
       <ProjectGeneralSettings params={{orgId: org.slug, projectId: project.slug}} />,
-      routerContext
+      {context: routerContext}
     );
-    expect(wrapper.find('Switch[name="scrapeJavaScript"]').prop('isDisabled')).toBe(true);
-    expect(wrapper.find('Switch[name="scrapeJavaScript"]').prop('isActive')).toBeFalsy();
+
+    expect(getField('checkbox', 'Enable JavaScript source fetching')).toBeDisabled();
+    expect(getField('checkbox', 'Enable JavaScript source fetching')).not.toBeChecked();
   });
 
-  it('project admins can remove project', async function () {
+  it('project admins can remove project', function () {
     const deleteMock = MockApiClient.addMockResponse({
       url: `/projects/${org.slug}/${project.slug}/`,
       method: 'DELETE',
     });
 
-    wrapper = mountWithTheme(
+    render(
       <ProjectGeneralSettings params={{orgId: org.slug, projectId: project.slug}} />
     );
 
-    const removeBtn = wrapper.find('.ref-remove-project').first();
-
-    expect(removeBtn.prop('children')).toBe('Remove Project');
-
-    // Click button
-    removeBtn.simulate('click');
+    userEvent.click(screen.getByRole('button', {name: 'Remove Project'}));
 
-    // Confirm Modal
-    modal = await mountGlobalModal();
-    modal.find('Button[priority="danger"]').simulate('click');
+    // Click confirmation button
+    renderGlobalModal();
+    userEvent.click(screen.getByTestId('confirm-button'));
```

Comment:
Will probably go back in the future and replace using the `within` strategy. We have a mix of both right now